src/state/nftMarket/hooks.ts

```diff
@@ -97,3 +99,8 @@ export const useGetNFTInitializationState = () => {
 export const useUserNfts = (): UserNftsState => {
   return useSelector((state: State) => state.nftMarket.data.user)
 }
+
+export const useHasGen0Nfts = (): boolean => {
+  const userNfts = useSelector((state: State) => state.nftMarket.data.user)
+  return userNfts.nfts.some((nft) => Number(nft.tokenId) <= MAX_GEN0_ID)
```

Comment:
`tokenId` is not Gen0 identifier, it is ordinal "mint" number of this bunny. You need to check `nft.attributes[0].value`.
For PancakeBunnies `attribute` field looks like this:
```
[
  {
      traitType: 'bunnyId',
      value: 3,
      displayType: null,
  }
]
```
so you take 0st attribute and `value` is the bunnyId number

Reply:
Oh I wasn't aware of that. Will change thanks.